tests/org.jboss.tools.cdi.bot.test/src/org/jboss/tools/cdi/bot/test/wizard/template/ProjectWithCDITemplate.java

```diff
@@ -209,26 +205,44 @@ public void createUtilityProject() {
 		uw.open();
 		UtilityProjectFirstPage up = new UtilityProjectFirstPage(uw);
 		up.setProjectName(PROJECT_NAME);
-		up.activateFacet("1.8", "Java");
+		if (isJavaLE8() || CDIVersion.equals("1.0") || CDIVersion.equals("1.2")) {
+			up.activateFacet("1.8", "Java");
+		} else {
+			up.activateFacet("11", "Java");
+		}
 		uw.finish();
+		new WaitUntil(new JobIsRunning(), TimePeriod.DEFAULT, false);
+		new WaitWhile(new JobIsRunning(), TimePeriod.LONG);
```

Comment:
these two waits are suspicious, can you please look at this closer? If you need both waits, then please add comments

Reply:
Fixed.